### DISCUSSION TITLE
p/answer-anotherdayu/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#347] [General]
# p/answer-anotherdayu/

简单介绍下自己或者你的博客？ 大家好，我是 Meng小羽，也是 Debug客栈 的博主。这是一个陪伴了我整整八年的博客平台。\n最初创办这个网站时，我只是希望用它来记录大学期间的学习笔记，以及算法竞赛中的刷题心得。随着时间推移，博客逐渐成长为一个涵盖多领域的技术分享平台，内容范围也从单一的技术积累拓展到更多元化的话题。\nDebug客栈 目前已成为一间“全能型的杂货铺”，在这里你可以看到：\n技术积累与分享：深入探讨服务端开发、前沿科技等技术内容； 科技趣闻与产品体验：分享数码产品的试用体验与评测； 热点讨论：关注时事，畅谈社会热点与见解； 好物与软件推荐：推荐高效实用的软件和工具，为你的生活和工作增色。 目前，我的职业是 服务端开发工程师，专注于系统设计、性能优化以及服务架构相关领域。对于我的更详细介绍，欢迎移步到：关于站长 - Debug客栈。\n感谢你关注 Debug客栈，也期待与大家在这里共同成长，探索更多有趣、有价值的内容！\n什么契机让你开始写博客？ 我的大学专业是 计算机科学与技术，相比其他同龄人，我较早接触到了互联网的思想。从学习编程开始，我遇到不懂的知识点或问题时，总是习惯通过 Google 搜索相关答案和解题思路。在这个过程中，我逐渐接触到了个人博客这一领域。\n让我印象最深刻的是互联网早期的浪潮，当时人们习惯搭建个人博客，最火的当属新浪博客。而我上大学时，微信公众号迅速崛起，但作为一个计算机相关专业的学生，拥有一个属于自己的主页或站点，不仅能积累知识与技术栈，更是一件非常值得骄傲的事情。\n于是，我开始了自己的博客之旅。从第一篇博文 《Sublime Text：崇高的文本编辑器》 开始，一发不可收拾。之后，各种学习笔记和竞赛相关的文章便陆续在博客上发表，为我的知识积累之路画上了一笔又一笔精彩的注脚。\n你是如何完成创作的？ 最初，我的站点使用的是 WordPress，通过 LNMP（Linux、Nginx、MySQL、PHP）架构进行部署和发布。当时的写作流程很简单：我会先在后台创建好文章标题，根据优先级安排博文的书写，同时反复打磨内容和语言组织。当然，也有不少文章因为各种原因“鸽”了 😄。\n到了 2023 年，我的博客完成了 全站静态化升级，采用了 Hugo 静态化生成的方式，并部署在 Cloudflare 上。写作工具方面，我曾用 VSCode 写过一段时间的博文，但总感觉不太适合写作，于是转向了 Obsidian。现在，我的流程是：先在 Obsidian 中创建文章标题，完成初稿后再润色一遍，最后复制到 VSCode 进行发布。\n此外，博客内容也会同步发布到微信公众号「Debug客栈」，欢迎关注！\n运营博客的过程中是否有失去过动力？ 如果有，是为什么恢复的？如果没有，请问您又是如何保持创作的激情？\n在运营博客的这 8年 里，我从未想过要放弃。相反，我对折腾充满热情。在这段时间里，我几乎每年都会更换一次博客主题，站点也从动态站点迁移到了静态站点。同时，我始终坚持更新博客文章，虽然 月更 对我来说有点难 😅，但 季更 还是稳稳地保持着。随着时间的推移，我不断提升自己的写作水平，也逐渐积累了一定的影响力。目前，全网关注人数已接近 2万。\n对我来说，博客不仅是一个记录的平台，更是展示技术能力与技术影响力的窗口。作为一名互联网从业者，我始终相信博客的价值：它不仅让我系统性地梳理知识，还帮助我结识了许多志同道合的伙伴。在这个过程中，我持续学习，不断成长。\n如何搭建博客，以及运营博客每年需要投入的资金？ 目前，我的博客搭建既简单又省心，但需要具备一定的编程基础。以下是我发布一篇博文的完整流程，与前面提到的创作步骤有些相似：\n在 Obsidian 书写草稿：先完成博文的初步内容； AI 润色与校对：使用 AI 工具润色博文，并检查错别字； 上传文章到 GitHub：博客文章存储在 GitHub 仓库，我通过 VSCode 将文章上传； 替换内部超链接：在 VSCode 中替换超链接，方便文章内链； 提交代码并合并分支：将文章推送到 GitHub 的 dev 分支，随后提交合并到 main 分支； 触发 Hugo 编译：利用 Hugo Actions，将文章生成为静态网页； 部署到 Cloudflare Pages：通过 Cloudflare Pages 获取编译好的网页，并全球分发； 访问文章：完成以上步骤后，文章即可通过网站访问。 目前采用的部署方式非常经济高效。借助 GitHub 和 Cloudflare Pages，博客仓库的部署额度和全球分发额度完全免费。唯一的开销是域名注册费用，每年在 ¥80 左右，可以说是高性价比的博客运营方案。\n

https://blog.debuginn.com/p/answer-anotherdayu/

**Comment by DayuGuo** [comment]
看了你的文章才知道Sublime也能装插件！

**Reply by debuginn** [reply]
大学的时候还经常用，现在只用 VSCode 了

**Comment by kanghaov** [comment]
厉害，顶

**Reply by debuginn** [reply]
感谢支持 🫰

**Comment by uuanqin** [comment]
我也是使用obsidian写博客，文本保存在本地才安心

**Reply by debuginn** [reply]
是滴，现在云笔记关停的太多了，本身文章占用只在小部分，并且本地，也没有数据泄漏和安全问题。
